share-through-htb.js

```diff
@@ -199,6 +120,44 @@ function ShareThroughHtb(configs) {
         }
     }
 
+    function __generateAdm(bid, placementKey) {
+        var stxResponseName = "str_response_" + bid.bidId;
+        var encodedBid = __b64EncodeUnicode(JSON.stringify(bid));
```

Comment:
IS the purpose here to hash the bid json? We are noticing an error with the wrapper with the encodedBid it generates and throws an error since the string below is not properly formatted